requirements.txt

```diff
@@ -4,102 +4,248 @@
-toml==0.10.0              # via tox
-tox==3.14.3               # via -r requirements.in
-tqdm==4.41.1              # via -r ./requirements-package.in
-traitlets==4.3.3          # via ipython
-urllib3==1.25.7           # via requests
-virtualenv==16.7.9        # via tox
-wcwidth==0.1.8            # via prompt-toolkit, pytest
-whichcraft==0.6.1         # via cookiecutter
-zipp==1.0.0               # via importlib-metadata
+alabaster==0.7.12
+    # via sphinx
+anytree==2.8.0
+    # via -r ./requirements-package.in
+argparsetree==0.0.6
+    # via -r ./requirements-package.in
+arrow==0.15.5
+    # via jinja2-time
+attrs==19.3.0
+    # via
+    #   hypothesis
+    #   jsonschema
+    #   pytest
+babel==2.8.0
+    # via sphinx
+backcall==0.1.0
+    # via ipython
+billiard==3.6.0.0
+    # via -r ./requirements-package.in
+binaryornot==0.4.4
+    # via cookiecutter
+certifi==2019.11.28
+    # via
+    #   -r ./requirements-package.in
+    #   requests
+chainmap==1.0.3
+    # via -r ./requirements-package.in
+chardet==3.0.4
+    # via
+    #   -r ./requirements-package.in
+    #   binaryornot
+    #   requests
+click==7.0
+    # via
+    #   cookiecutter
+    #   pip-tools
+configparser==4.0.2
+    # via -r requirements.in
+cookiecutter==1.7.0
+    # via -r ./requirements-package.in
+coverage==5.0.3
+    # via
+    #   -r requirements.in
+    #   pytest-cov
+decorator==4.4.1
+    # via
+    #   ipython
+    #   traitlets
+discover==0.4.0
+    # via -r requirements.in
+docutils==0.16
+    # via sphinx
+entrypoints==0.3
+    # via flake8
+filelock==3.0.12
+    # via tox
+flake8==3.7.9
+    # via -r requirements.in
+freezegun==0.3.13
+    # via -r requirements.in
+future==0.18.2
+    # via
+    #   -r ./requirements-package.in
+    #   -r requirements.in
+    #   cookiecutter
+hypothesis==5.1.5
+    # via -r requirements.in
+idna==2.8
+    # via requests
+imagesize==1.2.0
+    # via sphinx
+ipdb==0.12.3
+    # via -r requirements.in
+ipython-genutils==0.2.0
+    # via traitlets
+ipython==7.11.1
+    # via ipdb
+jedi==0.15.2
+    # via ipython
+jinja2-time==0.2.0
+    # via cookiecutter
+jinja2==2.10.3
+    # via
+    #   cookiecutter
+    #   jinja2-time
+    #   sphinx
+jsonschema==3.2.0
+    # via -r ./requirements-package.in
+llvmlite==0.35.0
+    # via numba
+markupsafe==1.1.1
+    # via jinja2
+mccabe==0.6.1
+    # via flake8
+mock==3.0.5
+    # via -r requirements.in
+more-itertools==8.1.0
+    # via pytest
+msgpack==1.0.0
+    # via -r ./requirements-package.in
+numba==0.52.0
+    # via -r ./requirements-package.in
+numexpr==2.7.1
+    # via -r ./requirements-package.in
+numpy==1.18.1
+    # via
+    #   numba
+    #   numexpr
+    #   pandas
+packaging==20.0
+    # via
+    #   pytest
+    #   sphinx
+    #   tox
+pandas==1.1.0
+    # via -r ./requirements-package.in
+parameterized==0.7.1
+    # via -r requirements.in
+parso==0.5.2
+    # via jedi
+pathlib2==2.3.5
+    # via -r ./requirements-package.in
```

Comment:
If you're in the modd to remove some more Python 2 stuff, you should be able to just remove this, and replace `from pathlib2 import...` to `from pathlib import...`

Reply:
Thats a good shout, I've give it a sweep, thanks!